utils/cloneByDate.py

```diff
@@ -46,21 +51,27 @@
     from common.rhnConfig import CFG, initCFG
     from satellite_tools.progress_bar import ProgressBar
 
-from depsolver import DepSolver
+from builtins import input
```

Comment:
Do we have `builtins` provided for both Python 2 and 3 ?

Reply:
This was a leftover. I have removed the line since it was not needed.